### DISCUSSION TITLE
Can't set window_animation_duration event if I gave screen record permission

### DISCUSSION THREAD

**Discussion by Max0u** [asmvik/yabai#1650] [Q&A]
Hello !

I've set window_animation_duration like following:
```
                window_animation_duration  0.15         \
```

Even if I allowed the screen recording I still get `command 'window_animation_duration' for domain 'config' requires Screen Recording permissions! ignoring request..` 

Did I miss something ?

**Comment by devidw** [comment]
Same:

<img width="903" alt="Screenshot 2024-03-23 at 10 07 13 AM" src="https://github.com/koekeishiya/yabai/assets/68775926/89f212b3-4002-4765-a333-2c5a78cab741">

**Reply by devidw** [reply]
Edit: **sign out & sign in again fixed it**

However I am still facing animation delays when switching between full screen apps, but maybe there is another config option to archive instant full screen window switches?

**Reply by devidw** [reply]
Edit: After opting out of macOS beta program and running latest updates to 14.4 (23E214) it works without any transitions/animation slow-downs ✨

I think yabai itself was not working properly before these os updates, since switching spaces with focus was not doing anything
